tests/fakeredis/pyproject.toml

```diff
@@ -37,10 +38,11 @@ markers = [
     "min_server",
     "max_server",
     "decode_responses",
+    "unsupported_server_types",
 ]
 asyncio_mode = "strict"
 generate_report_on_test = true
-render_collapsed = "failed,error"
+render_collapsed = "faipoled,error"
```

Comment:
is it correct @cunla  ?
all these changes are intended? (just double checking)

Reply:
No, good catch! fixed.